app/src/main/java/ru/appintheair/flightreviewssubmit/SubmitScreen.kt

```diff
@@ -0,0 +1,167 @@
+package ru.appintheair.flightreviewssubmit
+
+import android.graphics.drawable.Drawable
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
```

Comment:
It's possible to change rating bars while the progress is running
[video.mp4.zip](https://github.com/Pank-Off/FlightsApp/files/5537975/video.mp4.zip)